PaperRock.js

```diff
@@ -0,0 +1,85 @@
+/* Rock, Paper, or Scissors
+Rock paper scissors is a classic two player game. Each player chooses either rock, paper, or scissors. The items are compared, and whichever player chooses the more powerful item wins.
+
+The possible outcomes are:
+
+Rock destroys scissors.
+Scissors cut paper.
+Paper covers rock.
+If there's a tie, then the game ends in a draw.
+Our code will break the game into four parts:
+
+Get the user's choice.
+Get the computer's choice.
+Compare the two choices and determine a winner.
+Start the program and display the results. */
+
+
+const getUserChoice = function(userInput){
+    userinput = userInput.toLowerCase();
+    if (userInput === 'rock') {
+      return userInput;
+    }else if (userInput === 'paper') {
+      return userInput;
+    }else if (userInput === 'scissors'){ 
+        return userInput;
+    } else {
+      console.log('Wrong choice');
+    }
+  }
+  
+  console.log(getUserChoice('paper'));
+  console.log(getUserChoice('fork'));
+  
+  const randomNumber = Math.floor(Math.random() *3);
+  
+  const getComputerChoice = function(randomNumber) {
+    if (randomNumber === 0) {
+      return 'paper';
+    }else if (randomNumber === 1) {
+      return 'rock';
+    }else if (randomNumber === 2){ 
+        return 'scissors';
+    } else {
+      console.log('Wrong choice');
```

Comment:
Tu tak samo jak wyżej, tylko zrób array z liczbami

Reply:
możesz nawet zrobić oddzielną funkcję, która będzie sprawdzała czy jest to odpowiedni wybór, z tym, że to co daje user musisz zamienić ze stringa na numery (patrz obiekty)